### DISCUSSION TITLE
Export package.json for builds

### DISCUSSION THREAD

**Discussion by khesayed** [keycloak/keycloak#35776] [Adapter deprecation]
package.json file not added to exports this breaks module federation builds for microfrontend architecture.

Is it okay to open a new PR for this fix?

![Screenshot 2024-12-10 at 10 03 10 PM](https://github.com/user-attachments/assets/e8352e38-939f-41a5-955c-1045e54a0d1e)

**Comment by jonkoops** [comment]
As mentioned under https://github.com/keycloak/keycloak/pull/35779#issuecomment-2535468228, the `package.json` is not part of the public API of the package, this is intentional. If this breaks module federation builds this is a bug in whatever build system is used, it should not be reading package files using the Node.js module resolution (e.g. `require()` or `import`), but instead it should read them as files using the Filesystem API.
